webapp/src/components/answerValue/__tests__/MultiFileValue.test.js

```diff
@@ -16,21 +16,17 @@ describe('MultiFileValue Tests', () => {
   });
 
   it('Renders list of files.', () => {
-    const value = `[{"id":1,"file":"1.tst","name":"lorem.tst"},{"id":2,"file":"2.tst","name":"ipsum.tst"}]`
+    const value = `[{"id":1,"filename":"1.tst","rename":"lorem.tst"},{"id":2,"filename":"2.tst","rename":"ipsum.tst"}]`
```

Comment:
made changes here to resolve this failing test

Reply:
Awesome thank you!